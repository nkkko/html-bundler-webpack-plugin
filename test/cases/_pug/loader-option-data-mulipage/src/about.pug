html
  head
    title About
  body
    h1= title.create('customized')
    div Param[a]: #{a}
    div Param[b]: #{b}
