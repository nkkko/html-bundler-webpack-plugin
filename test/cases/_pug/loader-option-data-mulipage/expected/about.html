<!DOCTYPE html><html><head><title>About</title></head><body><h1>About 'customized' title!</h1><div>Param[a]: xyz</div><div>Param[b]: 987</div></body></html>